src/features/NewOrder/NewOrder.js

```diff
@@ -0,0 +1,300 @@
+          })}
+        ></input>
+      </div>
+
+      <div>
+        {errors.quantity && errors.quantity.type === 'required' && (
+          <div className={styles.error}>Quantity must be selected</div>
+        )}
+        {errors.quantity && errors.quantity.type === 'validate' && (
+          <div className={styles.error}>
+            Cash remaining must be $0 or greater
+          </div>
+        )}
+        <label htmlFor="quantity">Quantity</label>
+
+        <input
+          className={errors.quantity ? styles.inputError : styles.input}
+          name="quantity"
+          type="number"
+          step="1"
+          min="1"
+          ref={register({
+            required: true,
+            validate: (value) => value * sharePrice.current < totalCash,
+          })}
+        ></input>
+      </div>
+
+      <div>
+        {errors.price && errors.price.type === 'required' && (
+          <div className={styles.error}>Price must be selected</div>
+        )}
+        <label htmlFor="price">Order Share Price</label>
+        <input
+          className={errors.price ? styles.inputError : styles.input}
+          name="price"
+          min="0.01"
+          type="number"
+          step="0.01"
+          ref={register({ required: true })}
+        ></input>
+      </div>
+
+      <hr className={styles.hr}></hr>
+
+      <div>
+        <div className={styles.tally}>
+          Total Price: ${totalPrice.toFixed(2)}
+        </div>
+      </div>
+      <div>
+        {errors.cashRemaining && errors.cashRemaining.type === 'validate' && (
+          <div className={styles.error}>
+            You don't have enough cash remaining
+          </div>
+        )}
+        <div className={styles.tally}>
+          {' '}
+          Cash Remaining: ${cashRemaining.toFixed(2)}
+        </div>
+      </div>
+
+      <input type="submit" className={styles.submit} />
+    </form>
+  );
+}
+
+function SellForm() {
+  // variables
+  const { register, handleSubmit, watch, reset, errors } = useForm({
+    mode: 'onBlur',
+    reValidateMode: 'onBlur',
+  });
+
+  const onSubmit = (data) => {
+    console.log(data);
+    reset();
+  };
+
+  const onErrors = (errors) => console.error(errors);
+
+  const stock = useRef({});
+  stock.current = watch('stock');
+
+  const sharePrice = useRef({});
+  sharePrice.current = watch('price', '0.00');
+
+  const quantity = useRef({});
+  quantity.current = watch('quantity', '');
+
+  let totalPrice = sharePrice.current * quantity.current;
+
+  // functions
+  let optionsArray = [];
+  function options() {
+    for (var i = 0; i < ownedStocks.length; i++) {
+      optionsArray[i] = <option>{ownedStocks[i].stockName}</option>;
+    }
+  }
+  options();
+
+  function quantityCheck() {
+    for (var i = 0; i < ownedStocks.length; i++) {
+      if (ownedStocks[i].stockName === stock.current) {
+        return ownedStocks[i].stockShares;
+      } else {
+        return 999999;
```

Comment:
If you need a really large number you may consider using `Infinity` or `Number.MAX_SAFE_INTEGER` (depending on your specific use case)